release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -1516,6 +1516,35 @@ if you configured the `HostedCluster.spec.configuration.oauth` field, this setti
+
+* Previously, after installing an Azure Red Hat OpenShift cluster, some cluster Operators were unavailable. This was the result of one of the cluster’s load balancers not being created as part of the installation process. With this update, the load balancer is correctly created. After installing a cluster, all cluster Operators are available. (link:https://issues.redhat.com/browse/OCPBUGS-24191[*OCPBUGS-24191*])
+
+* Previously, if the VMware vSphere cluster contained an ESXi host that was offline, the installation failed with a "panic: runtime error: invalid memory address or nil pointer dereference" message. With this update, the error message states that the ESXi host is unavailable. (link:https://issues.redhat.com/browse/OCPBUGS-20350[*OCPBUGS-20350*])
+
+* Previously, if you only used the default machine configuration to specify existing AWS security groups when installing a cluster on AWS (`platform.aws.defaultMachinePlatform.additonalSecurityGroupsIDs`), the security groups were not applied to control plane machines. With this update, existing AWS security groups are correctly applied to control planes when they are specified using the default machine configuration. (link:https://issues.redhat.com/browse/OCPBUGS-20525[*OCPBUGS-20525*])
+
+* Previously, installing a cluster on AWS failed when the specified machine instance type (`platform.aws.type`) did not support the machine architecture that was specified for control plane or compute machines (`controlPlane.architecture` and `compute.architecture`). With this update, the installation program now checks to determine if the machine instance type supports the specified architecture and displays an error message if it does not. (link:https://issues.redhat.com/browse/OCPBUGS-26051[*OCPBUGS-26051*])
+
+* Previously, the installation program did not validate some configuration settings before installing the cluster. This behavior occurred when these settings were only specified in the default machine configuration (`platform.azure.defaultMachinePlatform`). As a result, the installation would succeed even if the following conditions were met:
++
+* An unsupported machine instance type was specified.
+* Additional functionality, such as accelerated networking or the use of Azure ultra disks, was not supported by the specified machine instance type.
++
```

Comment:
I think you need the following:
```suggestion
** An unsupported machine instance type was specified.
** Additional functionality, such as accelerated networking or the use of Azure ultra disks, was not supported by the specified machine instance type.
+
```
because:
![Screenshot 2024-02-21 at 10 02 50 AM](https://github.com/openshift/openshift-docs/assets/14860856/f4812a9c-aea8-4b2b-a2d2-c7673a68732f)

Reply:
@jldohmann Changed, how does it look now?

Reply:
the final paragraph is getting connected to the sub-bullet:
![Screenshot 2024-02-21 at 12 24 22 PM](https://github.com/openshift/openshift-docs/assets/14860856/f34856d5-7bc6-46d3-9849-3c21d6a6763c)

try removing the `+` on line 1543

Reply:
Hmmm, there is no `+` on that line, that's from the github interface.

Reply:
sorry i meant 1541 😭